app/assets/stylesheets/newflow_layout.scss

```diff
@@ -18,82 +44,346 @@ body {
 #signinup-form {
     max-width: 75rem;
     border-top: 0;
-    margin-top: 3rem;
     margin-left: auto;
     margin-right: auto;
+    margin-bottom: 3rem;
+    margin-top: 4rem;
+    padding-left: 14px;
+    padding-right: 14px;
+
+    .step-counter {
+        height: 30px;
+
+        > div {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            width: 94px;
+            height: 100%;
+
+            background-color: #f5f5f5;
+            border-radius: 1.5rem;
+        }
+
+        span {
+            font-size: 1.4rem;
+            font-weight: bold;
+            letter-spacing: -0.56px;
+            // padding: 1rem;
+        }
+    }
+
+    .tab-group {
+        padding: 0;
+
+        .tab {
+            @include width-up-to($phone-max) {
+                width: 50%;
+            }
+
+            width: 18rem;
+            font-size: 1.8rem;
+            font-weight: 700;
+
+            border-bottom: 0.1rem solid #d5d5d5;
+            padding: 1.2rem 0;
+            border-style: solid;
+            border-color: #d5d5d5;
+            border-width: 0.1rem;
+
+            &:first-child {
+                border-right: 0;
+            }
+
+            &:last-child {
+                width: 10rem;
+            }
+
+            &.active {
+                border-bottom-width: 0;
+                border-bottom-color: #fff;
+            }
+
+            &.inactive {
+                background-color: #f6f8fa;
+            }
+        }
+
+        .tab.tab--placeholder {
+            @include width-up-to($phone-max) {
+                display: none;
+            }
+
+            cursor: default;
+            border: 0;
+            border-bottom: 0.1rem solid #d5d5d5;
+            background-color: #fff; // TODO: same as html or body background color OR transparent
+        }
+    }
 
     form {
+        @include width-up-to($phone-max) {
+            row-gap: 2rem;
+            padding-left: 1.4rem;
+            padding-right: 1.4rem;
+            padding-top: 1.6rem;
+        }
+
+        @include wider-than($phone-max) {
+            row-gap: 1rem;
+            padding-top: 3rem;
+        }
+
         border-color: #d5d5d5;
         border-style: solid;
         border-width: 0.1rem;
         border-top: 0;
+        border-bottom-left-radius: .3rem;
+        border-bottom-right-radius: .3rem;
+
+        font-size: 1.4rem;
 
         h1 {
-            font-size: 36px;
+            @include width-up-to($phone-max) {
+                font-size: 18px;
```

Comment:
Definitely use `rem` for font sizes. And you can use `@include set-font(h4)`. That takes care of the `letter-spacing` as well.

Reply:
Done.